otel/metrics.go

```diff
@@ -0,0 +1,78 @@
+package otel
+
+import (
+	"strconv"
+
+	"github.com/itzg/mc-monitor/utils"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+const (
+	serverHostAttribute    = "server_host"
+	serverPortAttribute    = "server_port"
+	serverEditionAttribute = "server_edition"
+	serverVersionAttribute = "server_version"
+)
+
+type serverMetrics struct {
+}
+
+func Metrics() *serverMetrics {
+	return &serverMetrics{}
+}
+
+func (m *serverMetrics) RecordHealth(healthy bool, attributes []attribute.KeyValue) {
+	_ = NewInt64ObservableGauge(
```

Comment:
I'm thinking it's better not to return the gauge from the new functions if it's always ignored.

Reply:
Makes sense. We're not using for now and I think it will not have any use in the future!